app/src/main/java/com/example/sharewhatyoucanproject/home/HomeFragment.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.sharewhatyoucanproject.home
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.fragment.findNavController
+import com.example.sharewhatyoucanproject.databinding.FragmentHomeBinding
+
+class HomeFragment : Fragment() {
+    private var _binding: FragmentHomeBinding? = null
+    private val binding get() = _binding!!
```

Comment:
make sure to null out the binding in `onDestroyView`

Reply:
done